src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -15982,6 +15982,11 @@ typedef NS_OPTIONS(uint32_t, MTRLevelControlFeature) {
     MTRLevelControlFeatureFrequency = 0x4,
 };
 
+typedef NS_OPTIONS(uint8_t, MTRLevelControlOptions) {
+    MTRLevelControlOptionsExecuteIfOff = 0x1,
+    MTRLevelControlOptionsCoupleColorTempToLevel = 0x2,
+};
```

Comment:
This needs an MTR_NEWLY_AVAILABLE annotation....  If you're not comfortable with sorting out how to do that, please make sure an issue is filed, labeled "darwin", assigned to me.

Reply:
Sorry I don't know how to make that annotation (or where I should add it).
I have created an issue, but I don't have permission to add labels nor assign it. Issue created: https://github.com/project-chip/connectedhomeip/issues/23680

Reply:
Thank you, that's perfect.